database/migrations/2021_03_23_202453_create_metar_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMetarTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('metars', function (Blueprint $table) {
+            $table->id();
+            $table->text('raw')->comment('The raw METAR text');
+            $table->unsignedInteger('airfield_id')->unique()->comment('The airfield the METAR applies to');
+            $table->unsignedInteger('qnh')
```

Comment:
Would there be any merit of storing this in a JSON column so you wouldn't have to add a new column for each kind of new data (I'm possibly thinking of something like LVP detection etc?)

Reply:
67e5a27fcad1742e752df8ccf8d112b058755805

Reply:
Love it!